client/styles/components/_copyable-input.scss

```diff
@@ -50,4 +51,15 @@
     color: getThemifyVariable('button-background-hover-color');
     border-top-color: getThemifyVariable('button-background-hover-color');
   }
+}
+
+/*============================
+== Media queries
+============================*/
+
+@media only screen and (max-width: 775px) {
```

Comment:
could you make this media query into a SASS mixin? and put the width 775 into a variable? https://medium.com/codeartisan/breakpoints-and-media-queries-in-scss-46e8f551e2f2

Reply:
sure